app/views/restaurants/new.html.haml

```diff
@@ -3,30 +3,4 @@
     - @restaurant.errors.full_messages.each do |message|
```

Comment:
One last thing here .. 😉 
You might want to condition this to only display if we actually have error messages. That's the reason why you currently have that empty `{}` above the form.